storage/client.go

```diff
@@ -295,6 +296,19 @@ func (c Client) exec(verb, url string, headers map[string]string, body io.Reader
 	}
 
 	req, err := http.NewRequest(verb, url, body)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
As in `errors.New("storage.Client: error creating request object: " + err.Error())`?